src/main.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+import VueCookie from 'vue-cookie';
+import axios from 'axios';
+import VueAxios from 'vue-axios';
+
+import FavStar from './FavStar.vue';
+import EditIdentitiesForm from './EditIdentitiesForm.vue';
+
+const instance = axios.create({
+  xsrfCookieName: 'csrftoken',
+  xsrfHeaderName: 'X-CSRFToken',
+  baseURL: '/',
+  withCredentials: true,
+});
+
+Vue.config.productionTip = false;
+Vue.use(VueCookie);
+Vue.use(VueAxios, instance);
+
+Array.from(document.getElementsByClassName('favstar')).forEach(el => {
+  const data = {
+    active: el.dataset.active !== 'false',
+    username: el.dataset.username,
+  };
+  new Vue({
+    el,
+    render: createElement => createElement(FavStar, { props: data }),
+  });
+});
+
+new Vue({
```

Comment:
I'd be tempted to wrap this in the same `Array.from(document.getElements…` setup as you have above, so that `new Vue` is only evaluated if the edit form is present in the page, just to limit the risk of side effects, but that's based on intuition rather than a reasoned analysis of the code.

Reply:
As it stands, because of how `$mount` works, it'll be instantiated no matter what, but only attached if the element exists. I can see an argument for not bothering to do the work, even just the setup work, if it's not there though.

Reply:
On consideration, not worth it until we have measurable performance concerns.